.github/scripts/install_yugabyte.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Copyright (c) YugaByte, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied.  See the License for the specific language governing permissions and limitations
+# under the License.
+
+set -eo pipefail
+export YBC_HOME_DIR=`dirname $(which $0)`
+source $YBC_HOME_DIR/common.sh
+
+YB_VERSION_BUILD=$1
```

Comment:
Check that required parameters were passed in before using them.

Reply:
done checking for existence of the variable.